Makefile

```diff
@@ -325,6 +325,7 @@ KIND_EXPORT_LOGS ?=/tmp/kind_logs
 .PHONY: kind-export-logs
 kind-export-logs: create-export-logs
 	$(LOCALBIN)/kind export logs --name ${KIND_CLUSTER_NAME} ${KIND_EXPORT_LOGS}
+	$(LOCALBIN)/kubectl cluster-info dump --output-directory ${KIND_EXPORT_LOGS} --namespaces $(NAMESPACE)
```

Comment:
what does this do? Can you expand the commit message?

Reply:
The k8s reported didn't help me in 
https://github.com/zeeke/openperouter/actions/runs/19700867838/job/56436727346

because the a BeforeAll node was failing.
I think this commit is a little off topic for this PR, I'll propose an improvement to the failure reporting system in a different work.

Dropping the commit